bot.py

```diff
@@ -41,15 +43,49 @@ def pastebin(command):
     if response["status_txt"] == "OK":
         return "output truncated; full output at: " + response["data"]["url"]
     else:
+        print(response)
+        return "failed to shorten url"
+
+def gist(version, code):
+    url = "https://api.github.com/gists"
+
+    r = requests.post("https://api.github.com/gists",
+            json = {
+                "description": "Shared via Rust Playground",
+                "public": True,
+                "files": {
+                    "playbot.rs": {
+                        "content": code
+                    }
+                }
+            })
+
+    if r.status_code == 201:
+        response = r.json()
+
+        gist_id = response["id"]
+        gist_url = response["html_url"]
+
+        play_url = "https://play.rust-lang.org/?gist=" + quote(gist_id) + "&version=" + version
+        return "output truncated; full output at: " + play_url
+    else:
         return "failed to shorten url"
 
-def evaluate(code, nickname):
-    if nickname == "rusti" or nickname == "playbot":
-        version, _ = playpen.execute("stable", "/bin/dash",
+def pastebin(channel, code):
+    if channel == "nightly":
+        return gist(channel, code)
+    else:
+        return bitly(code)
```

Comment:
For now let's stick to just bitly links and leave gisting for features to a separate PR